### PR TITLE
eliminate 17 antecedents in polynomial bag theorems

### PR DESCRIPTION
**Pull request by icecream17**
and
* elrabi, elrn*, eldm* ax-10,11,12 reductions
* `g`-suffixed variants of theorems with ` F e. V ` instead of ` I e. V ` to avoid ax-rep (only a handful of ax-rep usages saved... it's likely that that the theorems I add here can be used more, also most of the df-map section can have versions avoiding ax-rep, but that's out of the scope of this pr)

commit by commit (though a few things are out of order)

Axiom changes: https://github.com/metamath/set.mm/commit/6f430cd8583700f75c3a512dcd88539172cc33a3 (does not include savings from ssrab2)

---

I added a revision tag and OLD version if I removed an antecedent, this gets repetitive but it seems to be the process: https://github.com/metamath/set.mm/pull/4101

(if nothing changed except the proof uses a newer theorem, I don't add tags or OLD versions)

### CODE REVIEW

File: set.mm
Comment (benjub) [inline set.mm:33984]:
```suggestion
       variable condition on ` A , x ` and avoid ~ ax-10 , ~ ax-11 ,
```
(That's how I write it for the HTML rendering with, I think, less spacing.)

File: set.mm
Comment (benjub) [inline set.mm:54633]: Probably there is no `y` in these two proofs ?

File: set.mm
Comment (benjub) [inline set.mm:264128]:
```suggestion
    $( Finite bags have finite support.  (Contributed by Stefan O'Rear,
```

File: set.mm
Comment (benjub) [inline set.mm:264115]: It could be "Remove a sethood antecedent.", to give a bit more information. Since sethood antecedents are quite common in set.mm, this gives a good idea of the nature of the revision.  (The fact that it was extraneous is contained in the fact that you could remove it.)

File: set.mm
Comment (benjub) [inline set.mm:264225]: Shorten **proof** and...

File: set.mm
Comment (benjub) [inline set.mm:264530]: Cannot the curly braces be adjusted now (possibly by having shorter scopes, which is often a better practice anyway since long scopes are error-prone), so that we do not forget it next year when we remove the OLD versions ?

**Review by benjub — APPROVED** [review]
LGTM, but I only skimmed through the later changes, so another pair of eyes would be appreciated (the second approver could also be the merger).

File: set.mm
Comment (avekens) [inline set.mm:264548]: Here, I think it is a sethood hypothesis which was removed.

**Review by benjub — APPROVED** [review]

**Review by avekens — APPROVED** [review]

**Pull request merged**
